### PR TITLE
Fixing Hang Issue in KeyFile Integration Test

### PR DESCRIPTION
**Pull request by raj-prince**
**Issue:** By default when we pass wrong key-file it retries several time and returns the response after ~200 secs. We have two test-cases in that particular test (KeyFile), so it returns the response after ~400 secs, which feels like the hang in the integration test.

**Fix:**    By setting the --max-retry-sleep to 0, we only try to connect to the server once which also removes the concept of wait time and we are able to complete the test in 4-5 secs.

### CODE REVIEW

File: tools/integration_tests/mounting/gcsfuse_test.go [hunk 1]
```diff
@@ -21,7 +21,6 @@ import (
 	"os"
 	"os/exec"
 	"path"
-
```
Comment (avidullu) [inline tools/integration_tests/mounting/gcsfuse_test.go:24]: Is this formatting done by some IDE?

File: tools/integration_tests/mounting/gcsfuse_test.go [hunk 2]
```diff
@@ -194,12 +191,13 @@ func (t *GcsfuseTest) KeyFile() {
 	}{
 		// Via flag
 		0: {
-			extraArgs: []string{fmt.Sprintf("--key-file=%s", nonexistent)},
+			extraArgs: []string{fmt.Sprintf("--key-file=%s", nonexistent), "--max-retry-sleep=0"},
```
Comment (avidullu) [inline tools/integration_tests/mounting/gcsfuse_test.go:194]: Add a comment as to why this max-retry-sleep flag is needed.

File: tools/integration_tests/mounting/gcsfuse_test.go (hunk 1 above)
Comment (raj-prince) [reply tools/integration_tests/mounting/gcsfuse_test.go:24]: yes.

File: tools/integration_tests/mounting/gcsfuse_test.go (hunk 2 above)
Comment (raj-prince) [reply tools/integration_tests/mounting/gcsfuse_test.go:194]: done.

Comment (avidullu) [reply tools/integration_tests/mounting/gcsfuse_test.go:194]: I don't see the comment. Do you need to "commit something" ?

Comment (raj-prince) [reply tools/integration_tests/mounting/gcsfuse_test.go:195]: Ohh, I updated the comment in description part, now it's done.

**Review by avidullu — APPROVED** [review]

**Pull request merged**